tests/ui/static_ref.stderr

```diff
@@ -0,0 +1,11 @@
+error[E0597]: `_pool` does not live long enough
+ --> $DIR/static_ref.rs:9:1
+  |
+9 | #[pymethods]
+  | ^^^^^^^^^^^-
+  | |          |
+  | |          `_pool` dropped here while still borrowed
+  | borrowed value does not live long enough
+  | cast requires that `_pool` is borrowed for `'static`
+  |
+  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

Comment:
Looks like this `note` was added in early feb: https://github.com/rust-lang/rust/commit/26c86a6a2828b1048fa2fd425595ef8fb475249a

So might need to bump minimum rust version slightly to keep travis happy?

Reply:
Yeah, but since to find the minimum nightly by binary search is really hard, I just skip the test using `TRAVIS_JOB_NAME`.